styleguide/source/_patterns/03-organisms/search-page-filters.twig

```diff
@@ -0,0 +1,9 @@
+<h1>Search Results</h1>
+
+<form class="joe__search-bar">
+  <label class="visually-hidden">Site Search</label>
```

Comment:
Surprised that there isn't placeholder text.

Reply:
screen readers sometimes don't read placeholder text, so it is good to have a label for accessibility reasons. But i'll add placeholder too.